xskillscore/core/probabilistic.py

```diff
@@ -484,4 +488,134 @@ def discrimination(
         dim=dim,
     ) / (xr.ufuncs.logical_not(observations)).sum(dim=dim)
```

Comment:
this line throws a warning, depreciated once xr doesnt support np 1.17

Reply:
Changed to `np.logical_not()`